DAO.sol

```diff
@@ -691,6 +700,62 @@ contract DAO is DAOInterface, Token, TokenCreation {
         paidOut[msg.sender] = 0;
         return true;
     }
+    
+    function splitSoloDAO(bool _confirm
+    ) noEther onlyTokenholders returns (bool _success) {
+        
+        // Ensure user confirms solo split and is not blocked
+        if (!_confirm || isBlocked(msg.sender))
+            throw;
+        
+        // Create new Solo DAO based off this contract
+        SplitData memory soloData;
```

Comment:
Probably don't need to create a SplitData object. In the original splitDAO() it was necessary because you wanted to take a snapshot of the data so that if subsequent users join the split DAO, they will have the original split data since the data have changed from the first person that split.